bin/update

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env ruby
 require 'pathname'
 require 'fileutils'
-include FileUtils
 
-# path to your application root.
 APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
 
 def system!(*args)
   system(*args) || abort("\n== Command #{args} failed ==")
 end
 
 chdir APP_ROOT do
-  # This script is a way to update your development environment automatically.
-  # Add necessary update steps to this file.
+  include FileUtils
```

Comment:
@marderer is it rubocop who forced you to move `include FileUtils` from the beginning of the file? If yes - let's disable this cop and move `include FileUtils` back at the start of the file. If no - describe why have you done it?

Reply:
It is rubocop